ccdaservice/ssmanager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+$ignoreAuth = true;
+// require_once(dirname(__file__) . './../interface/globals.php');
+require_once ( dirname( __FILE__ ) . "/../library/log.inc" );
+require_once ( dirname( __FILE__ ) . "/../library/sql.inc" );
+set_time_limit( 0 );
+ob_implicit_flush();
+session_write_close(); // Release session lock to prevent freezing of other scripts
+ignore_user_abort( 1 );
```

Comment:
I forgot about them. They are hold over from my php socket server when I had them integrated together. Code it once and forget about it, that's my motto.:) They can go.